modules/agar/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+### Added
+- Added `TestStore` API, which was moved from Alloy. #TINY-9157
```

Comment:
I could have sworn I left a suggestion for this but I can't find it sorry. Anyways, maybe something like this to clarify it also needs types.

```suggestion
- Added `TestStore` API, which was moved from Alloy and requires types. #TINY-9157
```